src/main/java/gov/usgs/earthquake/nats/NATSStreamingNotificationReceiver.java

```diff
@@ -180,7 +151,16 @@ public void onMessage(Message message) {
       // update sequence and tracking file if exception not thrown or we still want to update sequence anyway
       if (!exceptionThrown || updateSequenceAfterException) {
         sequence = message.getSequence();
-        writeTrackingFile();
+        trackingFile.write(
```

Comment:
Same as WebsocketNotificationReceiver, also watch indentation.

Reply:
Added readTrackingFile/writeTrackingFile methods local to NATSStreamingNotificationReceiver, watched indentation